libs/clients/district-commissioners-licenses/src/lib/districtCommissionersLicenses.provider.ts

```diff
@@ -0,0 +1,33 @@
+import { Configuration, RettindiFyrirIslandIsApi } from '../../gen/fetch'
+import { Provider } from '@nestjs/common'
+import {
+  ConfigType,
+  IdsClientConfig,
+  LazyDuringDevScope,
+} from '@island.is/nest/config'
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+
+export const DistrictCommissionerLicensesApiProvider: Provider<RettindiFyrirIslandIsApi> =
+  {
+    provide: RettindiFyrirIslandIsApi,
+    scope: LazyDuringDevScope,
+    useFactory: (idsClientConfig: ConfigType<typeof IdsClientConfig>) =>
+      new RettindiFyrirIslandIsApi(
+        new Configuration({
+          fetchApi: createEnhancedFetch({
+            name: 'clients-district-commissioners-licenses',
+            organizationSlug: 'syslumenn',
+            autoAuth: idsClientConfig.isConfigured
+              ? {
+                  mode: 'tokenExchange',
+                  issuer: idsClientConfig.issuer,
+                  clientId: idsClientConfig.clientId,
+                  clientSecret: idsClientConfig.clientSecret,
+                  scope: [],
```

Comment:
Using an empty scope is IDS pitfall, as IDS will create an access token with all the scopes that the clientId has access to. And given that this is used in the GQL api this will be the gql machine client which has a lot of 3rd party scopes, the sýslumenn api will get a rather big access token which they could use to access other 3rd party services. 

So we would want to use some scope here.

Reply:
ah yeah ofc, will fix

Reply:
Done ✅